README.md

```diff
@@ -39,12 +39,16 @@ Commit your code regularly and use descriptive messages. This helps both you (in
 Demonstrate your understanding of this week's concepts by answering the following free-form questions. Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager.
 
 - [ ] What is the purpose of using _sessions_?
+    To make sure that the information stays the same for each new request instead of reentering each time
 
```

Comment:
the main purpose for using sessions is to be certain that the person who is looking at the registered user's view is the same as the one who logged in. Everytime a request is made, the cookie from the front-end will be compared to the logged in user's cookie that was added on that initial login on the backend. If it doesn't match, some error handling is needed to log the account out.